swagger.yaml

```diff
@@ -1355,6 +1355,26 @@ components:
         status:
           $ref: '#/components/schemas/TaskStatus'
           description: Task status. TaskConnections is taken into account when updating the status
+        assigneeIds:
+          type: array
+          items:
+            type: string
+          description: List of assignee user IDs
+        userRole:
+          $ref: '#/components/schemas/UserRole'
+          description: User role suggested for the task
+        estimatedDuration:
+          type: string
+          description: Estimated duration of the task
+        estimatedReadiness:
+          type: string
+          description: Estimated readiness of the task
```

Comment:
This does not depend on task start and end dates, right?

Reply:
No, those are just user defined valuse, so probably string is good enough